clients/admin-ui/cypress/e2e/systems.cy.ts

```diff
@@ -1,15 +1,15 @@
 import { stubSystemCrud, stubTaxonomyEntities } from "cypress/support/stubs";
 
-// TODO: Update Cypress test to reflect the nav bar 2.0
-describe.skip("System management page", () => {
```

Comment:
this whole test suite was being skipped, when really we only had to skip the nav bar related one (see lines 11/12 below)